### PR TITLE
Enable Mellanox kernel configs

### PR DESCRIPTION
**Pull request by chalamalasetty**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/tools/cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [ ] Ready to merge

---

###### Summary 

What does the PR accomplish, why was it needed?
  Enable Mellanox kernel configs for SR-IOV support in CBL-Mariner.

###### Change Log  

- Change
- Change
- Change

###### Does this affect the toolchain?  

**YES**
NO

###### Associated issues  

- #xxxx

###### Links to CVEs  
- https://nvd.nist.gov/...

###### Test Methodology

- Pipeline build id: xxxx

### CODE REVIEW

File: SPECS/kernel/kernel.spec [hunk 1]
```diff
@@ -2,7 +2,7 @@
 Summary:        Linux Kernel
 Name:           kernel
 Version:        5.4.51
-Release:        5%{?dist}
+Release:        6%{?dist}
```
Comment (christopherco) [inline SPECS/kernel/kernel.spec:5]: Please also bump the Release number and add a changelog entry in `SPECS-SIGNED/kernel-signed-x64` and `SPECS-SIGNED/kernel-signed-aarch64`

**Review by christopherco — CHANGES_REQUESTED** [review]

File: SPECS/kernel/kernel.spec (hunk 1 above)
Comment (chalamalasetty) [reply SPECS/kernel/kernel.spec:5]: Thank you. I will do the needful.

Comment (chalamalasetty) [reply SPECS/kernel/kernel.spec:5]: Added

**Review by christopherco — APPROVED** [review]

**Comment by chalamalasetty** [comment]
Rebase image to resolve merge conflicts ended up with issues. Hence closing this PR and creating a new PR for changes to merge.
https://github.com/microsoft/CBL-Mariner/pull/142

**Pull request closed without merging**